tests/puppeteer/campaigns/functional/BO/03_catalog/02_categories/04_changeCategoryPosition.js

```diff
@@ -0,0 +1,113 @@
+require('module-alias/register');
+// Using chai
+const {expect} = require('chai');
+const helper = require('@utils/helpers');
+const loginCommon = require('@commonTests/loginBO');
+// Importing pages
+const BOBasePage = require('@pages/BO/BObasePage');
+const LoginPage = require('@pages/BO/login');
+const DashboardPage = require('@pages/BO/dashboard');
+const CategoriesPage = require('@pages/BO/catalog/categories');
+// Test context imports
+const testContext = require('@utils/testContext');
+
+const baseContext = 'functional_BO_catalog_categories_changeCategoryPosition';
+
+let browser;
+let page;
+let numberOfCategories = 0;
+
+// Init objects needed
+const init = async function () {
+  return {
+    boBasePage: new BOBasePage(page),
+    loginPage: new LoginPage(page),
+    dashboardPage: new DashboardPage(page),
+    categoriesPage: new CategoriesPage(page),
+  };
+};
+
+describe('Change category position', async () => {
+  // before and after functions
+  before(async function () {
+    browser = await helper.createBrowser();
+    page = await helper.newTab(browser);
+    this.pageObjects = await init();
+  });
+  after(async () => {
+    await helper.closeBrowser(browser);
+  });
+
+  // Login into BO and go to categories page
+  loginCommon.loginBO();
+
+  it('should go to categories page', async function () {
+    await testContext.addContextItem(this, 'testIdentifier', 'goToCategoriesPage', baseContext);
+    await this.pageObjects.boBasePage.goToSubMenu(
+      this.pageObjects.boBasePage.catalogParentLink,
+      this.pageObjects.boBasePage.categoriesLink,
+    );
+    await this.pageObjects.boBasePage.closeSfToolBar();
+    const pageTitle = await this.pageObjects.categoriesPage.getPageTitle();
+    await expect(pageTitle).to.contains(this.pageObjects.categoriesPage.pageTitle);
+  });
+
+  it('should reset all filters and get Number of Categories in BO', async function () {
```

Comment:
```suggestion
  it('should reset all filters and get number of categories in BO', async function () {
```
Otherwise on line 44 should be Categories? wdyt?

Reply:
You are right for this one, all step's description should be in lower cases (for all it).
And for describes, sentence should start with an upper case

Reply:
fixed